README.md

```diff
@@ -2,53 +2,76 @@
 
 [![Gitter](https://img.shields.io/gitter/room/nwjs/nw.js.svg)](https://gitter.im/strongloop/loopback) [![Build Status](https://travis-ci.org/strongloop/loopback-next-quick-start.svg?branch=master)](https://travis-ci.org/strongloop/loopback-next-quick-start) [TODO Code coverage - coveralls]
 
-Welcome to the quick start tutorial for LoopBack.next + TypeScript!
-LoopBack makes it easy to build modern applications that require complex integrations. LoopBack.next is the next version of LoopBack and is being designed to make it even easier to extend the framework for your own needs and will enable the latest ES features.
+## Table of Contents
+* [Overview](#overview)
+* [Setup](#setup)
+* [Testing](#testing)
+* [Project Structure](#structure)
+* [TypeScript Configuration](#typescriptConfiguration)
+* [Dependencies](#dependencies)
+* [Other Resources](#resources)
+* [Contribute](#contribute)
 
-Goal of tutorial
-Show you what a LoopBack.next app looks like and how the basics works
+## <a name="overview"></a>Overview
 
-what we're trying to do
-start a server to serve content
-make a request to the server to fetch content
+#### Welcome to the quick start tutorial for Loopback.next + TypeScript!
+Loopback makes it easy to build modern applications that require complex integrations. 
 
-how we do it
-create a server
-add a custom component
-add a custom sequence
+Loopback.next is the next version of LoopBack and is being designed to make it even easier to extend the framework for your own needs and will enable the latest ES features.
```

Comment:
Once merged @emckean will work on README.  I've communicated this point with @emckean .